### PR TITLE
Update EasyRSA version

### PR DESCRIPTION
**Pull request by robiiinos**
EasyRSA 3.0.7 has been tagged few days ago.

https://github.com/OpenVPN/easy-rsa/releases/tag/v3.0.7

### CODE REVIEW

**Review by randshell — CHANGES_REQUESTED** [review]
They changed the filename again...
Can you please update it?

**Comment by randshell** [comment]
Fixes #603

**Comment by TinCanTech** [comment]
> They changed the filename again...

This is how EasyRSA always make releases.
Take a look here:
https://github.com/OpenVPN/easy-rsa/releases

**Comment by angristan** [comment]
They changed the filenames.

**Comment by TinCanTech** [comment]
Ah yes, on this occasion they have actually changed the file names.

There has been quite a lot a dev work done (by me)

**Comment by TinCanTech** [comment]
From what I know, I believe the filenames should be _stable_ now.

**Comment by angristan** [comment]
Hopefully!

**Comment by angristan** [comment]
Thanks @robiiinos

**Comment by randshell** [comment]
> Ah yes, on this occasion they have actually changed the file names.

Filenames aren't stable from release 3.0.4

File: openvpn-install.sh [hunk 1]
```diff
@@ -681,8 +681,8 @@ function installOpenVPN() {
 	# Install the latest version of easy-rsa from source, if not already
 	# installed.
 	if [[ ! -d /etc/openvpn/easy-rsa/ ]]; then
-		local version="3.0.6"
-		wget -O ~/EasyRSA-unix-v${version}.tgz https://github.com/OpenVPN/easy-rsa/releases/download/v${version}/EasyRSA-unix-v${version}.tgz
+		local version="3.0.7"
+		wget -O ~/EasyRSA-unix-v${version}.tgz https://github.com/OpenVPN/easy-rsa/releases/download/v${version}/EasyRSA-${version}.tgz
 		tar xzf ~/EasyRSA-unix-v${version}.tgz -C ~/
 		mkdir -p /etc/openvpn/easy-rsa
 		mv ~/EasyRSA-v${version}/* /etc/openvpn/easy-rsa/
```
Comment (randshell) [inline openvpn-install.sh:688]: @angristan This should be `mv ~/EasyRSA-${version}/* /etc/openvpn/easy-rsa/` without the `v`

Comment (angristan) [reply openvpn-install.sh:688]: https://github.com/angristan/openvpn-install/commit/0e961a2e6b362574c705d00b121fbecad4d1de83 :)

**Comment by TinCanTech** [comment]
> > Ah yes, on this occasion they have actually changed the file names.
> 
> Filenames aren't stable from release 3.0.4

Yes but they should be now. A lot of development has gone into EasyRSA since 304.

**Pull request merged**